modules/aws/dra-analytics/variables.tf

```diff
@@ -82,8 +82,8 @@ variable "archiver_user" {
 
 variable "dra_version" {
   type        = string
-  default     = "4.13"
-  description = "The DRA version to install. Supported versions are 4.11.0.10.0.7 and up. Both long and short version formats are supported, for example, 4.11.0.10.0.7 or 4.11. The short format maps to the latest patch."
+  default     = "4.14"
+  description = "The DRA version to install. Supported versions are 4.11.0.10 and up. Both long and short version formats are supported, for example, 4.11.0.10 or 4.11. The short format maps to the latest patch."
```

Comment:
There's a bug in the description, the modules accept only full version format, it's neither long nor short. So it should be:
"The DRA version to install. Supported versions are 4.11.0.10.0.7 and up. Full version format is supported, for example, 4.11.0.10.0.7."